### PR TITLE
Bugfix/934

### LINKED ISSUE #934: file list is not updated after file/folder rename
### Steps to reproduce
1. rename a file or folder

### Expected behaviour
the file list should be updated with the new name

### Actual behaviour
file list stays the same, only updated after a reload

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
Fix behavior upon renaming

## Related Issue
- Fixes #934 

## How Has This Been Tested?
- :hand: 
- :robot: 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2316/

File: tests/acceptance/features/webUIRenameFiles/renameFiles.feature [hunk 1]
```diff
@@ -9,7 +9,6 @@ Feature: rename files
     And the user has browsed to the files page
 
   @smokeTest
-  @issue-934
   Scenario Outline: Rename a file
     When the user renames file "lorem.txt" to <to_file_name> using the webUI
     #Then file "<to_file_name>" should be listed on the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIRenameFiles/renameFiles.feature]: need to be enabled

File: tests/acceptance/features/webUIRenameFolders/renameFolders.feature [hunk 2]
```diff
@@ -8,7 +8,6 @@ Feature: rename folders
     And user "user1" has logged in using the webUI
     And the user has browsed to the files page
 
-  @issue-934
   Scenario Outline: Rename a folder
     When the user renames folder "simple-folder" to <to_folder_name> using the webUI
     #Then folder <to_folder_name> should be listed on the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIRenameFolders/renameFolders.feature]: need to be enabled

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2321/

**Comment by individual-it** [comment]
the screenshots look like :robot: is clicking on the download button and not the rename button :thinking:

**Comment by DeepDiver1975** [comment]
> the screenshots look like robot is clicking on the download button and not the rename button thinking

let me double check the button ids ....

**Comment by DeepDiver1975** [comment]
should be fine ...

Screenshot from 2019-04-15 16-31-32

**Comment by DeepDiver1975** [comment]
the issue is here
```
    ✖ failed
      Error while running "waitForElementVisible" command: Timed out while waiting for element <#files-breadcrumb li:nth-of-type(1)> to be present for 10000 milliseconds. - expected "visible" but got: "not found"
          at Proxy.waitForElementVisible (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/filesPage.js:13:10)
at World.navigateToFolder (/var/www/owncloud/phoenix/tests/acceptance/stepDefinitions/filesContext.js:34:34)
```

breadcrumb is off screen -> not visible -> failing

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2326/

**Comment by DeepDiver1975** [comment]
and 

```
   ? Then file ""'quotes2'"" should be listed on the webUI
       Undefined. Implement with the following snippet:

         Then('file {string}{string}{string} should be listed on the webUI', function (string, string2, string3) {
           // Write code here that turns the phrase above into concrete actions
           return 'pending';
         });

   - When the user reloads the current page of the webUI # tests/acceptance/stepDefinitions/filesContext.js:83
- Then file "'quotes2'" should be listed on the webUI # tests/acceptance/stepDefinitions/filesContext.js:63

**Comment by individual-it** [comment]
> ```
>     ✖ failed
>       Error while running "waitForElementVisible" command: Timed out while waiting for element <#files-breadcrumb li:nth-of-type(1)> to be present for 10000 milliseconds. - expected "visible" but got: "not found"
>           at Proxy.waitForElementVisible (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/filesPage.js:13:10)
> at World.navigateToFolder (/var/www/owncloud/phoenix/tests/acceptance/stepDefinitions/filesContext.js:34:34)
> ```

opening a folder just does not work, it shows the download screen - same for me locally - works fine on master

openfolder

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2332/

**Comment by individual-it** [comment]
> `- Then file "'quotes2'" should be listed on the webUI # tests/acceptance/stepDefinitions/filesContext.js:63`

there have been too many `"` in the line. I've deleted them

**Pull request merged**